### ISSUE TITLE
failed

### ISSUE THREAD

**Issue by oddrobbia** [ManlyMarco/KK_PantyFairy#6]
Hello can you add a slider with the percent chance for success i spaming the steal buttom and so common its failed

**Comment by ManlyMarco** [comment]
It should succeed almost every time if the heroine is a lover.

**Comment by oddrobbia** [comment]
yes but if the girl is not a lower just a friend in the story mode i commonly stay with some girl as a sex friend and i tryed 50 times in a row to steal the clothes/panties from the girl and at the finaly it worked very common its failed and they notice i use the cheat menu to reset the conversation time every times

**Comment by ManlyMarco** [comment]
Well, the idea is that they are more cautious around you, and depending on the character traits it might get really difficult since those are taken into account as well. As far as I can see it's working as intended, you just happened to find a character that's very resistant.

**Comment by oddrobbia** [comment]
can you at least put a small information about whose character personality or trait are resistant to the steal because i dont know how its works as a heroine is friend very common the failue

**Comment by ManlyMarco** [comment]
https://github.com/ManlyMarco/KK_PantyFairy/blob/main/src/KKS_PantyFairy/Functions/PantyStealFeat.cs#L459
